### PR TITLE
Feature/expand event details

### PR DESCRIPTION
**Pull request by collinkallery**
### Type of change made:

- [X] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Styling (Styling to new or existing files)

### Detailed Description
- This PR includes conditional rendering for both the events container and event details to ensure that it does not render anything until the eventList / eventDetails is populated in the Redux store. 
- It also includes minor changes to styling for the eventContainer and buildout of the eventDetails component & styling. 
### Why is this change required? What problem does it solve?
- Allows a user to see more information for a given event 
- Conditionally renders to prevent errors 
### Was any previously written code changed?
- No

### CODE REVIEW

File: src/Actions/index.js [hunk 1]
```diff
@@ -10,3 +10,13 @@ export function getEvents() {
...
+
+export function findEvent(events, eventID) {
+  return dispatch => {
+    let foundEvent = events.filter(event => event.id === Number(eventID))
+    return dispatch({
+      type: 'FIND_EVENT',
+      payload: foundEvent[0]
+    });
+  }
+}
```
Comment (collinkallery) [inline src/Actions/index.js:22]: Reducer for finding an event to expand to that specific event's details.

File: src/App.js [hunk 2]
```diff
@@ -19,7 +21,11 @@ function App() {
       <Route exact path='/'component={SplashPage}/>
       <Route path='/home'component={EventContainer}/>
       <Route path='/create-event'component={CreateEvent}/>
-      <Route path='/my-events'component={MyEvents}/> 
+      <Route path='/my-events'component={MyEvents}/>
+      <Route path='/event-details/:eventID' render={({match}) => {
+        const { eventID } = match.params;
+        return <EventDetails eventID={eventID} />
+      }} />
```
Comment (collinkallery) [inline src/App.js:28]: Dynamic route for event details

File: src/Containers/EventContainer/EventContainer.js [hunk 3]
```diff
@@ -69,7 +71,7 @@ const EventContainer = (props) => {
         <SearchForm />
         <EventWrapper>
           <h2>Event<span>Feed</span></h2>
-          {!!props.eventList && displayEvents()}
+          {!!eventList && displayEvents()}
```
Comment (collinkallery) [inline src/Containers/EventContainer/EventContainer.js:74]: Conditional rendering to wait until the component has connected to the store.

File: src/Containers/EventDetails/EventDetails.js [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+    })
+    return performers;
+  }
+
+  const displayLabels = () => {
+    const labels = selectedEvent.labels.map((label, key) => {
+      return(<p key={key}>{label}</p>)
+    })
+    return labels;
+  }
```
Comment (collinkallery) [inline src/Containers/EventDetails/EventDetails.js:111]: Maps over performers and labels to display them

File: src/Containers/EventDetails/EventDetails.js [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+  justify-content: center;
+  width: 40%;
+
+  img {
+    width: 500px;
+    height: 500px;
+  }
+`
+
+const ButtonWrapper = styled.section`
```
Comment (danielbldr) [inline src/Containers/EventDetails/EventDetails.js:64]: I like this styling!

**Pull request merged**